distribution/kernel/carbon.product

```diff
@@ -13,7 +13,7 @@
    </plugins>
 
    <features>
-      <feature id="org.wso2.carbon.core.runtime" version="4.4.11"/>
+      <feature id="org.wso2.carbon.core.runtime" version="4.4.11.SNAPSHOT"/>
```

Comment:
Not having this fix simply cause build failure. How do you even to start a release perform without having a buildable source?

And during the release:perform it automatically change the version to 4.4.11 and then it will updated to 4.4.12.SNAPSHOT for the next iteration.

Can you please mention what is the change you are requesting?